### DISCUSSION TITLE
How to use Event processing with token

### DISCUSSION THREAD

**Discussion by tonyich** [Terrance/SkPy#280] [Q&A] [answered]
Please advise, how to use Event processing with saving the token in a file when restarting the script?
I use sample code from https://skpy.t.allofti.me/guides/events.html

**Comment by tonyich** [comment]
My code:
```
if __name__ == "__main__":
	sk = MySkype(tokenFile='/userspace/tmp/skype.token', autoAck=True)
	if not (sk.skype.conn.connected):
		sk = MySkype("myLogin", "MyPassw", autoAck=True)
		sk.conn.writeToken()
	while True:
		sk.loop()
```

**Comment by Terrance** [comment, marked as answer]
Pass `tokenFile` to the [`SkypeEventLoop`](https://skpy.t.allofti.me/reference/api.html#skpy.main.SkypeEventLoop) constructor.
